libraries/ui/src/CookieBanner.tsx

```diff
@@ -5,6 +5,7 @@ import posthog from 'posthog-js';
 import { CTALinkOrButton } from './CTALinkOrButton';
 
 type CookieBannerProps = {
+  cookiePolicyUrl: string,
```

Comment:
I think this is _probably_ not worth parameterising. We only have one privacy policy across everything, so hard coding it within the component seems fine (possibly to bluedot.org/privacy-policy if we expect to use it across different apps and we're worried the relative path won't work)